doc/api/http.md

```diff
@@ -1677,8 +1685,9 @@ added: v0.3.6
 Node.js maintains several connections per server to make HTTP requests.
 This function allows one to transparently issue requests.
 
-`options` can be an object or a string. If `options` is a string, it is
-automatically parsed with [`url.parse()`][].
+`options` can be an object, a string, or a [`URL`][] object. If `options` is a string, it is
```

Comment:
If I get this right, `` [`URL`] `` should be added to the link references at the end of the file.